.github/workflows/run-tests.yml

```diff
@@ -38,12 +38,11 @@ jobs:
     - name: Install python packages
       shell: bash -el {0}
       run: |
-        # Note the pinning of zipp here is because versions 3.16.0+ do not support python 3.7. Specifying zipp can be entirely removed (will be pulled in by flake8) if pydm drops support for 3.7
         if [ "$RUNNER_OS" == "Windows" ]; then
-          mamba install flake8 zipp=3.15.0 pyqt=${{ matrix.pyqt-version }}
+          mamba install flake8 pyqt=${{ matrix.pyqt-version }}
```

Comment:
can references to flake8 be removed since code checking is done with pre-commit now?

Reply:
Good point, thanks! Removed remaining references to flake8 from test run file [647f7a9](https://github.com/slaclab/pydm/pull/1047/commits/647f7a9698d801cfafd1b09e60dacc34d989cda3)